### ISSUE TITLE
NVDA gets frozen permanently to the point where restarting NVDA is the only way to restore it

### ISSUE THREAD

**Issue by Neurrone** [nvaccess/nvda#14831]
### Actual behavior:

I'm not sure what triggers this. I noticed this happening twice so far, both happening when I played Hearthstone. I'm not sure if that's relevant, since to my knowledge the accessibility mod for it sends speech directly to NVDA.

An error sound is heard and speech suddenly stops completely, and when I use various commands to navigate in the Hearthstone game, nothing is heard. NVDA does not recover even after several minutes. The keyboard doesn't respond properly to key presses. 

Attached below is the error log from `%temp%/nvda-old.log`.

### Expected behavior:

No crashes.

### NVDA logs, crash dumps and other attachments:
[crash.log](https://github.com/nvaccess/nvda/files/11234186/crash.log)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-28043,14bc6bd2 (2023.2.0.28043)

#### Windows version:
Windows 10 Version 22H2 (OS Build 19045.2728)

#### Name and version of other software in use when reproducing the issue:
Hearthstone

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I saw it happening in a previous recent alpha, though I've only seen this a grand total of two times.

#### If NVDA add-ons are disabled, is your problem still occurring?
Unknown

**Comment by zstanecic** [comment]
Hi,

Can you test it on alpha-28089,9fe52071

 

 

From: Dickson Tan ***@***.***> 
Sent: Friday, April 14, 2023 6:33 PM
To: nvaccess/nvda ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [nvaccess/nvda] NVDA gets frozen permanently to the point where restarting NVDA is the only way to restore it (Issue #14831)

 


Steps to reproduce:


Actual behavior:


I'm not sure what triggers this. I noticed this happening twice so far, both happening when I played Hearthstone. I'm not sure if that's relevant, since to my knowledge the accessibility mod for it sends speech directly to NVDA.

An error sound is heard and speech suddenly stops completely, and when I use various commands to navigate in the Hearthstone game, nothing is heard. NVDA does not recover even after several minutes. The keyboard doesn't respond properly to key presses.

Attached below is the error log from %temp%/nvda-old.log.


Expected behavior:


No crashes.


NVDA logs, crash dumps and other attachments:


crash.log <https://github.com/nvaccess/nvda/files/11234186/crash.log> 


System configuration


NVDA installed/portable/running from source:


Installed


NVDA version:


alpha-28043,14bc6bd2 (2023.2.0.28043)


Windows version:


Windows 10 Version 22H2 (OS Build 19045.2728)


Name and version of other software in use when reproducing the issue:


Hearthstone


Other information about your system:


Other questions


Does the issue still occur after restarting your computer?


Yes


Have you tried any other versions of NVDA? If so, please report their behaviors.


I saw it happening in a previous recent alpha, though I've only seen this a grand total of two times.


If NVDA add-ons are disabled, is your problem still occurring?


Unknown


Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

**Comment by Neurrone** [comment]
I doubt that will make a difference, since I was only a couple of versions behind the latest snapshot.

I've since updated and will keep an eye out for it happening again.

**Comment by ABuffEr** [comment]
I'm quite sure it caused by my PR #14703, that I then asked to cancel  due to this freeze, absent in last alpha. So I suggest to close this issue and reopen if needed.

**Comment by Neurrone** [comment]
@ABuffEr the linked PR mentions freezes in the terminal, but I wasn't in a terminal when I encountered the freezes.

**Comment by ABuffEr** [comment]
> @ABuffEr the linked PR mentions freezes in the terminal, but I wasn't in a terminal when I encountered the freezes.

Yes, but it worked on dictionary processing side, and some checks to ensure you are in the terminal could caused the issue.

**Comment by Brian1Gaff** [comment]
If its any help, I have found that certain add ons can cause the eternal 
limbo effect since the 2023.1 update. These are supposedly good for the 
version, but seem to have got corrupted or their settings have. What I did 
is remove them and then manually remove the whole folder with them in, and 
then do a clean install. That then has fixed it. The actual add on almost 
seems random but might be more than 1. So do test with no add ons before you 
start pulling your hair out and reeenable them one at a time.
 Its one of those cases where its probably going to be impossible to trace 
the actual issue, but restarting  from a clean copy will work.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by seanbudd** [comment]
This seems related to #14518
